src/components/Table/Components/RowSelectionIndicator.tsx

```diff
@@ -1,11 +1,19 @@
 import { Event, Prop, Component, Base } from '@/core';
 import { Checkbox } from '@/components/Form';
```

Comment:
mixed usage of `@/` annotation and relative path

Reply:
fixed